screens/InventoryByCategoryScreen.js

```diff
@@ -39,17 +44,37 @@ export default class InventoryByCategoryScreen extends React.Component {
           inventory: inventory
         });
       }.bind(this));
+
+    await firebase
+    .database()
+    .ref('category/'+ this.state.category)
+    .once('value')
+    .then(function(data) {
+      var categoryVal = data.val();
+      this.setState({
+        categoryDisplayName: categoryVal['displayName']
+      });
+    }.bind(this));
   }
 
   renderInventory() {
     var itemRows = [];
     Object.entries(this.state.inventory).map(([itemName, itemCount]) => {
       itemRows.push(
-        <Text
-          style={styles.innerText}
-          key={itemName}>
-          {itemName} - Count: {itemCount}
-        </Text>
+        <TouchableOpacity 
+          key={itemName}
+          style={styles.touchable}>
+          <View style={styles.iconHolder}>
+            <Icon
+              name='food-apple-outline'
+              type='material-community'
+              size = {30} />
```

Comment:
Q: What is this doing? Do you have to do brackets around a number? :O

Reply:
yes lol